File: CSES/repetitions.java
import java.io.*;
import java.util.StringTokenizer;

//old java code ignore
public class repetitions {
    public static void main(String args[]) throws IOException
    {
        BufferedReader r = new BufferedReader(new InputStreamReader(System.in));
        PrintWriter pw = new PrintWriter(System.out);
        StringTokenizer st = new StringTokenizer(r.readLine());

        String test = st.nextToken();
        int max = 1;
        int count = 1;

        for (int j = 2; j <= test.length(); j++)
        {
            if(test.substring(j - 1, j).equals(test.substring(j - 2, j - 1)))
            {
                count += 1;
                if (count > max)
                {
                    max = count;
                }
                
            }
            else
            {
                count = 1;
            }
        }
        pw.println(max);

        pw.close();
    }
}

File: CSES/repetitions.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    cin.tie(0)->sync_with_stdio(0);

    int high = 1, count = 1;
    string test;

    cin >> test;

    for(int i = 1; i <= test.length(); i++)
    {
        if (test[i] != test[i - 1])
        {
            high = max(high, count);
            count = 0;
        }
        count++;
    }
    high = max(high, count);
    cout << high << "\n";
}